resource-managers/kubernetes/core/pom.xml

```diff
@@ -52,6 +52,18 @@
       <type>test-jar</type>
     </dependency>
 
+    <dependency>
+      <groupId>io.fabric8</groupId>
+      <artifactId>volcano-client</artifactId>
```

Comment:
Is this dependency used at the moment ?
I don't see an import of `io.fabric8.volcano.client` below.

Reply:
Yep, we already use kuberenetes-client directly, no needed.